### PR TITLE
[Mandatenbeheer] Fixing duplicate bestuursorgaan naam

### PR DESCRIPTION
**Pull request by Poltergeistz**
This reuses the same code from #205

Moving the `getUniqueBestuursorganen()` to mandataris class to reuse it in eredienst-mandatenbeheer.

### CODE REVIEW

File: app/models/mandataris.js [hunk 1]
```diff
@@ -20,3 +20,17 @@ export default class MandatarisModel extends AgentInPosition {
   @belongsTo('mandataris-status-code', { inverse: null }) status;
   @hasMany('contact-punt', { inverse: 'mandatarissen' }) contactPoints;
 }
+
+export async function getUniqueBestuursorganen(mandataris) {
```
Comment (Windvis) [inline app/models/mandataris.js:24]: I would also reuse this in the other module so the util code doesn't get out of sync.

Comment (Poltergeistz) [reply app/models/mandataris.js:24]: Yes, in this PR ?

Comment (Windvis) [reply app/models/mandataris.js:24]: Yea. In theory it could have been part of the work to move it to the model class.

**Review by Windvis — APPROVED** [review]
Thanks!

**Pull request merged**